src/sagemaker/amazon/amazon_estimator.py

```diff
@@ -611,6 +611,8 @@ def get_image_uri(region_name, repo_name, repo_version=1):
     """Return algorithm image URI for the given AWS region, repository name, and
     repository version
 
+    TODO: consider refactoring version/repo validation into a module for all frameworks
+
```

Comment:
I'm not sure I buy the usefulness of this comment here. We already have #1464.

Reply:
will remove